content/v2/domains/templates.markdown

```diff
@@ -0,0 +1,37 @@
+---
+title: Templates API | Domains | DNSimple API v2 (Beta)
+excerpt: This page documents the DNSimple templates API v2.
```

Comment:
Being picky I'd say something like: _DNSimple API v2 template documentation_ (avoiding the _This page_ part which makes me uneasy).